src/helpers/discord_utils.ts

```diff
@@ -474,7 +474,7 @@ export async function sendEndRoundMessage(messageContext: MessageContext,
         color = EMBED_ERROR_COLOR;
     }
 
-    const songAndArtist = bold(`"${gameRound.originalSongName}" - ${gameRound.artistName}`);
+    const songAndArtist = bold(`"${gameRound.originalSongName}" - ${gameRound.artistName}`.split("*").join("\\*"));
```

Comment:
Should probably fix this at the source, in the implementation of `bold`, and our other formatting functions

Reply:
s

Reply:
reset vote